tests/local/test_search.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import unicode_literals
+
+import unittest
+
+from mopidy.local import search
+from mopidy.models import Album, Track
+
+
+class LocalLibrarySearchTest(unittest.TestCase):
```

Comment:
Just as a quick FYI, the rest of local is tested via "integration" tests with core involved as well. It's actually been on the backlog to move much of the local tests hiding in core to files like this :-)